addons/bus/static/src/js/web_client_bus.js

```diff
@@ -0,0 +1,92 @@
+odoo.define('bus.WebClient', function (require) {
+"use strict";
+
+var core = require('web.core');
+var WebClient = require('web.WebClient');
+
+var _t = core._t;
+
+WebClient.include({
+    /**
+     * Detects the presence of assets in DOM's HEAD
+     *
+     * @override
+     */
+    start: function () {
+        var self = this;
```

Comment:
in master (and 13 actually), we can use ES6 + async/await. So here, I'd use async/await, an arrow function, and a `const` (or no variable):
```js
start: async function () {
    await this._super.apply(this, arguments);
    this._assetsChangedNotificationId = null;
    this._assets = {};
    document.querySelectorAll('*[data-asset-xmlid]').forEach(el => {
        this._assets[el.getAttribute('data-asset-xmlid')] = el.getAttribute('data-asset-version');
    });
});
```